prometheus_client/asgi.py

```diff
@@ -0,0 +1,34 @@
+from urllib.parse import parse_qs
+
+from .exposition import _bake_output
+from .registry import REGISTRY
+
+
+def make_asgi_app(registry=REGISTRY):
+    """Create a ASGI app which serves the metrics from a registry."""
+
+    async def prometheus_app(scope, receive, send):
```

Comment:
Will having these new keywords in the codebase cause problems for those still on Python 2?

Reply:
No, it should not cause any issues.

The `asgi.py` file is conditionally included in `exposition.py`, and hence will only be succesfully loaded in python3.6 and higher.
This was the primary argument for introducing this file, rather than just including this code directly in `exposition.py`.